s3tests_boto3/functional/test_s3select.py

```diff
@@ -1162,3 +1281,36 @@ def test_bool_cast_expressions():
     res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select count(*) from s3object where cast(_1 as int) != 0 ;')).replace("\n","")
 
     s3select_assert_result( res_s3select_cast, res_s3select )
+
+@attr('s3select')
+def test_progress_expressions():
+
+    csv_obj = create_random_csv_object(1000000,10)
```

Comment:
for better test coverage, the `csv_object` should contain several chunks (4m).
the last progress response should be validated, and not the first one(always the same).
the last progress-response depends on object size (random size)